File: app/src/main/java/com/bearcave/passageplanning/data/database/tables/waypoints/ReadWaypoints.java
package com.bearcave.passageplanning.data.database.tables.waypoints;

import java.util.List;

/**
 * Created by miwas on 20.05.17.
 */

public interface ReadWaypoints {
    List<WaypointDAO> read(List<Integer> ids);
}

File: app/src/main/java/com/bearcave/passageplanning/waypoints/database/ReadWaypoints.kt
package com.bearcave.passageplanning.waypoints.database

import com.bearcave.passageplanning.waypoints.database.Waypoint

/**
 *
 * @since 24.05.17.
 * @author Michał Wąsowicz
 * @version 1.0
 */

interface ReadWaypoints {
    fun readAllWaypoints(): List<Waypoint>
    fun readWith(ids: List<Int>): List<Waypoint>
}
